app/src/components/v-collection-field-template.vue

```diff
@@ -4,20 +4,21 @@ import type { Field, Relation } from '@directus/types';
 import { toRefs } from 'vue';
 import { useI18n } from 'vue-i18n';
 
-const props = defineProps<{
-	value: string | null;
-	placeholder?: string | null;
-	disabled?: boolean;
-	collection: string | null;
-	inject: {
-		fields: Field[];
-		relations?: Relation[];
-	} | null;
-}>();
+const props = withDefaults(
+	defineProps<{
+		modelValue?: string | null;
+		placeholder?: string | null;
+		disabled?: boolean;
+		collection: string | null;
+		inject?: {
+			fields: Field[];
+			relations?: Relation[];
+		} | null;
+	}>(),
+	{ inject: null },
```

Comment:
I think we could omit the default value, disallow `null` and just make it optional.
FWIW, the `inject` option isn't used anywhere at the moment - we might also remove it from this component and add it back if there's demand for it.